intermine/intermine
# InterMine 3.0.0 - Solr

## Details

Added Solr to InterMine. 

Please wait until 2.1.1 is published before merging.

Docs are here:

* [How to install SOLR](https://intermine.readthedocs.io/en/latest/system-requirements/software/solr/)
* [How to configure search results](https://intermine.readthedocs.io/en/latest/webapp/keyword-search/)

## Testing

* build works
* beta flymine
* beta humanmine
* thalemine - @sergiocontrino can you test?

* search should return similar results
* had issue with XML library. Make sure XML reading / writing works. (webapp, data loading and web services)

## Checklist

Before your pull request can be approved, be sure to check all boxes:

- [x] Passing unit test for new or updated code (if applicable)
- [x] Passes all tests – according to Travis
- [x] Single purpose
- [x] Well commented code
- [x] Checkstyle
